main.go

```diff
@@ -85,7 +86,8 @@ var (
 	vSphereClusterIdentityConcurrency int
 	vSphereDeploymentZoneConcurrency  int
```

Comment:
A bit unrelatd to this PR. can we fully implemented the profilerAddress flag (similar to core CAPI)

Right now we only have a var, but we don't have a flag and we don't pass it to the manager options

Reply:
Looks like we shouldn't have the var but we are setting the flag on ~L187

Reply:
Makes sense!